app/javascript/components/ResultsPanel.js

```diff
@@ -0,0 +1,119 @@
+  },
+  {
+    type:'study',
+    name: 'Single nucleus RNA-seq',
+    cell_count: 5426,
+    acession: 'SCP4',
+    description: 'orem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+  },
+  {type:'study',
+    name: 'Single nucleus RNA-seq',
+    cell_count: 5426,
+    acession: 'SCP5',
+    description: 'orem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+  },
+];
+
+class ResultsPanel extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      results: undefined,
+      facets : []
+    };
+    this.handleStudyLabel = this.handleStudyLabel.bind(this);
+    };
+
+    handleStudyLabel(studyURL){
+      // This eventually will be an API call that will render the associated
+      // Study page 
+      alert({studyURL})  
+    }
+    
+    showResults(){
+      // Function to hande future pagination
+    }
+    render(){
+      
+      return(
+        <div>
+          <Tabs defaultActiveKey='study' transition={false}>
+            <Tab eventKey='study' title="Studies" >
+              <StudyResults handleStudyLabel = {this.handleStudyLabel} results={this.state.results}/>
+            </Tab>   
+            <Tab eventKey='files' title='Files'/>       
+        </Tabs>
+        </div>
+  );
+  
+    }
+  }
+
+const StudyResults = (props) => {
+  // This component may have state/props where onMount a API call is made
+  // to get the initial results on the home page.
+  return(
+    <Tab.Content>
+       { props.results.length &&
+          props.results.map((result)=>(
+            <Study 
+              key={result.acession} 
+              study={result}
+              handleStudyLabel = {props.handleStudyLabel}
+              />
+          )
+              
```

Comment:
nice.  Do we have a message to display for when results.length is 0?  Something like 'no studies matched your criteria'?

Reply:
This can be taken care of in Keyword search and the Parent wrapper component. There are a couple of ways to approach this:

1.  In the event where the search API doesn't return results, then ResultsPanel shouldn't actually be rendered. Instead maybe a `<span> No Results</span>` or a `<p> No results</`>
2. When the `<ResultsPanel>` has not results to show it can render `<span> No Results</span>` or a `<p> No results</`>